README.md

```diff
@@ -1,29 +1,570 @@
 # json-ld.net
 
-A [JSON-LD][jsonld] processor for .NET.
-
 [![NuGet][nuget-badge]][nuget]
 ![Build Status][gha-badge]
 [![codecov][codecov-badge]][codecov]
 
-This project has adopted the [Microsoft Open Source Code of Conduct][coc].
-For more information see the [Code of Conduct FAQ][coc-faq] or contact
-[opencode@microsoft.com][ms-mail] with any additional questions or comments.
+## Introduction
+This library is an implementation of the JSON-LD specification in C#.
+
+JSON, as specified in RFC7159, is a simple language for representing objects on the Web. Linked Data is a way of describing content across different documents or Web sites. Web resources are described using IRIs, and typically are dereferencable entities that may be used to find more information, creating a "Web of Knowledge". JSON-LD is intended to be a simple publishing method for expressing not only Linked Data in JSON, but for adding semantics to existing JSON.
```

Comment:
Can we please keep the line length to maximum 80 characters? That makes the document easier to read as plain text.

Reply:
I think we need to strike a balance here between our personal preferences and discouraging contributions from others.

When considering what makes it easier "to read as plain text", I think it very much depends what you are using to access the text. Certain editors, terminals and shells do a good job of providing line wrapping facilities, and manual line wrapping can sometimes prevent users taking advantage of the facilities at their disposal. On the other hand there are the line-oriented editors and piping tools (I happen to be a fan of vi) which benefit from short discrete lines.

I don't mind doing the work but I do think we need to settle on a reasonably small set of well documented rules for contributors to follow. I would like to see _more_ community involvement in this project going forward if possible.

I'll go ahead and make the changes suggested, but can you think about putting together a short guide for contributors?

Reply:
Is the requirement to alter the code itself to adhere to a 80 character limit also? Some of these code examples will need a bit of an overhaul to fit within that narrow limit if so...

Reply:
same applies for the link references - I'm not sure how to keep those to 80 characters as the URLs are longer than that limit in some cases. Maybe there's some markdown black magic for this?

Reply:
ok i think this is close to what you wanted now